KSP-Backend/KSP/mvp/models.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from django.db import models
+
+YEAR_CHOICES = []
+for year in range(2016, (datetime.datetime.now().year + 1)):
+    YEAR_CHOICES.append((year, year))
+
+
+class Notes(models.Model):
+    course_id = models.CharField(max_length=10)
+    batch = models.IntegerField(choices=YEAR_CHOICES)
+    course_instructor = models.CharField(max_length=100)
+    uploaded_at = models.DateTimeField(auto_now=True)
+    gdrive_link = models.URLField(max_length=500)
+
+class Feedback(models.Model):
+    name = models.CharField(max_length=100)
+    course_id = models.CharField(max_length=10)
+    course_instructor = models.CharField(max_length=100)
+    batch = models.IntegerField(choices=YEAR_CHOICES)
+    created_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now=True)
+    feedback = models.TextField()
+
+class PYQs(models.Model):

```

Comment:
can you please keep the full name "past year question" PYQs are little confusing?

Reply:
ok

Reply:
> Few more points to note:
> 
> * Naming the app 'mvp' is not a good idea. The app name in the project should tell what the app's function is. For example, an app that deals with users can be named 'users', an app that is responsible for handling auth can be named 'auth'. Rename the app in your PR, please.
> * There is an already existing app named `dbmodel`. If it's not relevant to you, please delete it.

I deleted dbmodel app, actually mvp stands for minimum viable product.

Reply:
i did this